### DISCUSSION TITLE
DMA

### DISCUSSION THREAD

**Discussion by Radczenko** [olikraus/u8g2#2160] [General]
Hello,
  Do you thinking about implement DMA for i2c OLED ?

**Comment by JamToJum** [comment]
No, I am thinking of somehow implementing support for a monochrome display
of 128*64 pixels st7567s which is connected via the i2c bus

чт, 13 квіт. 2023 р. о 15:15 Radczenko ***@***.***> пише:

**Comment by olikraus** [comment]
> Hello, Do you thinking about implement DMA for i2c OLED ?

Using DMA for u8g2 in general would require to add a double buffer to u8g2, which is difficult.
